### ISSUE TITLE
Error in 'helpers.utils'

### ISSUE THREAD

**Issue by kekukui** [kodi-community-addons/script.module.metadatautils#32]
```
ERROR: script.module.metadatautils --> ERROR in helpers.utils ! --> ('Connection aborted.', BadStatusLine("''",))
WARNING: script.module.metadatautils --> Traceback (most recent call last):

File "/storage/.kodi/addons/script.module.metadatautils/lib/helpers/utils.py", line 114, in get_json
  response = requests.get(url, params=params, timeout=20)

File "/storage/.kodi/addons/script.module.requests/lib/requests/api.py", line 72, in get
  return request('get', url, params=params, **kwargs)

File "/storage/.kodi/addons/script.module.requests/lib/requests/api.py", line 58, in request
  return session.request(method=method, url=url, **kwargs)

File "/storage/.kodi/addons/script.module.requests/lib/requests/sessions.py", line 512, in request
 resp = self.send(prep, **send_kwargs)

File "/storage/.kodi/addons/script.module.requests/lib/requests/sessions.py", line 622, in send
  r = adapter.send(request, **kwargs)

File "/storage/.kodi/addons/script.module.requests/lib/requests/adapters.py", line 495, in send
  raise ConnectionError(err, request=request)

ConnectionError: ('Connection aborted.', BadStatusLine("''",))

ERROR: script.module.metadatautils --> ERROR in helpers.utils ! --> ('Connection aborted.',   BadStatusLine("''",))
```

**Comment by kekukui** [comment]
Another freeze possibly triggered by internet outage
```
WARNING: script.module.metadatautils --> Traceback (most recent call last):

 File "/storage/.kodi/addons/script.module.metadatautils/lib/helpers/utils.py", line 116, in get_json
result = json.loads(response.content.decode('utf-8', 'replace'))

 File "/storage/.kodi/addons/script.module.simplejson/lib/simplejson/__init__.py", line 538, in loads
return _default_decoder.decode(s)

 File "/storage/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 370, in decode
obj, end = self.raw_decode(s)

 File "/storage/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 400, in raw_decode
return self.scan_once(s, idx=_w(s, idx).end())

 File "/storage/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 79, in scan_once
return _scan_once(string, idx)

 File "/storage/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 70, in _scan_once
raise JSONDecodeError(errmsg, string, idx)

JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

`ERROR: script.module.metadatautils --> ERROR in helpers.utils ! --> Expecting value: line 1 column 1 (char 0)`

**Comment by ZedeN1** [comment]
Causes hard crashes with Kodi 18.7 running on Raspberry Pi 4B

``` php 
2020-11-25 03:38:07.688 T:667939024 WARNING: script.module.metadatautils --> Traceback (most recent call last):
                                              File "/home/pi/.kodi/addons/script.module.metadatautils/lib/helpers/utils.py", line 117, in get_json
                                                result = json.loads(response.content.decode('utf-8', 'replace'))
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/__init__.py", line 538, in loads
                                                return _default_decoder.decode(s)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 370, in decode
                                                obj, end = self.raw_decode(s)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 400, in raw_decode
                                                return self.scan_once(s, idx=_w(s, idx).end())
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 79, in scan_once
                                                return _scan_once(string, idx)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 70, in _scan_once
                                                raise JSONDecodeError(errmsg, string, idx)
                                            JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-11-25 03:38:07.688 T:667939024   ERROR: script.module.metadatautils --> ERROR in helpers.utils ! --> Expecting value: line 1 column 1 (char 0)
2020-11-25 03:38:08.996 T:740290768 WARNING: script.module.metadatautils --> Traceback (most recent call last):
                                              File "/home/pi/.kodi/addons/script.module.metadatautils/lib/helpers/utils.py", line 117, in get_json
                                                result = json.loads(response.content.decode('utf-8', 'replace'))
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/__init__.py", line 538, in loads
                                                return _default_decoder.decode(s)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 370, in decode
                                                obj, end = self.raw_decode(s)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 400, in raw_decode
                                                return self.scan_once(s, idx=_w(s, idx).end())
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 79, in scan_once
                                                return _scan_once(string, idx)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 70, in _scan_once
                                                raise JSONDecodeError(errmsg, string, idx)
                                            JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-11-25 03:38:08.996 T:740290768   ERROR: script.module.metadatautils --> ERROR in helpers.utils ! --> Expecting value: line 1 column 1 (char 0)
2020-11-25 03:38:09.170 T:740290768 WARNING: script.module.metadatautils --> Traceback (most recent call last):
                                              File "/home/pi/.kodi/addons/script.module.metadatautils/lib/helpers/utils.py", line 117, in get_json
                                                result = json.loads(response.content.decode('utf-8', 'replace'))
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/__init__.py", line 538, in loads
                                                return _default_decoder.decode(s)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 370, in decode
                                                obj, end = self.raw_decode(s)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/decoder.py", line 400, in raw_decode
                                                return self.scan_once(s, idx=_w(s, idx).end())
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 79, in scan_once
                                                return _scan_once(string, idx)
                                              File "/home/pi/.kodi/addons/script.module.simplejson/lib/simplejson/scanner.py", line 70, in _scan_once
                                                raise JSONDecodeError(errmsg, string, idx)
                                            JSONDecodeError: Expecting value: line 1 column 1 (char 0)
2020-11-25 03:38:09.171 T:740290768   ERROR: script.module.metadatautils --> ERROR in helpers.utils ! --> Expecting value: line 1 column 1 (char 0)
```

**Comment by Angelinas1** [comment]
update your script.module.simplejson from kodi repo
instal new version script.module.metadatautils and script.skin.helper.service from git hub
line from your log (line 117) now is 130....
https://github.com/kodi-community-addons/script.module.metadatautils/blob/master/lib/helpers/utils.py#L130

**Comment by ZedeN1** [comment]
Thanks, manually updating from github seems to have solved my issues!

My script.module.simplejson already was the latest available from kodi repo 3.16.1

script.module.metadatautils was newest available from repos 1.0.27  (github newest is 1.0.35)
script.skin.helper.service was newest available from repos 1.1.32 (github newest is 1.1.31)
